shopinvader_sale_exception/services/cart.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+
+
+class CartService(Component):
+    _inherit = "shopinvader.cart.service"
+
+    def check_exceptions(self, **params):
+        """Run sale exception checks"""
+        cart = self._get(create_if_not_found=False)
+        if not cart:
+            return {}
+        cart.detect_exceptions()
+        return self._to_json(cart)
```

Comment:
do we really have to return the whole cart?
IMO we should return only sale_exceptions here. Not a blocker tho.

Reply:
My POV:
1. this endpoint does not alter the cart, right?
2. this endpoint is called explicitly only to get this info, right?

Reply:
True. I've just pushed the changes 👍🏻 thanks!